### DISCUSSION TITLE
disable extra commit Notes added by 'git notes add' with jenkins integration

### DISCUSSION THREAD

**Discussion by y0zg** [semantic-release/semantic-release#2194] [Q&A]
I haven't faced this behaviour while integrating semantic release with gitlab, but after jenkins integration  I see an extra commit `Notes added by 'git notes add'`

so, I have 2 commits in pipeline

```
Notes added by 'git notes add'
chore(release): 1.0.10 [skip ci] 1.0.10 (2021-10-18) Bug Fixes test cicd (0f5e5bc) test ci
```

My config file 
```
plugins: [
  "@semantic-release/commit-analyzer",
  {
     "releaseRules": [
          {breaking: true, release: 'major'},
          {revert: true, release: 'patch'},
          {type: 'feat', release: 'minor'},
          {type: 'fix', release: 'patch'},
          {type: 'perf', release: 'patch'},
        ]
  },
  "@semantic-release/release-notes-generator",
  '@semantic-release/changelog',
  ["@semantic-release/git", {
      "assets": [
        "CHANGELOG.md",
      ]
    }
  ],
]
branches: [ 
  {name: "master",channel: "false"},
]
tagFormat: "${version}"
```

**Comment by y0zg** [comment]
found my answer in https://github.com/semantic-release/semantic-release/discussions/2017 but it's still unclear how to disable such behaviour. Asked the same question in bitbucket community https://community.atlassian.com/t5/Bitbucket-questions/disable-quot-git-notes-add-quot-behaviour-for-semantic-release/qaq-p/1837322#M66665

**Reply by y0zg** [reply]
Despite this issue is reproducible for Bitbucket, I suppose it's mostly related to semantic release rather than to Bitbucket itself as semantic release uses both `git tag` and `git note` https://semantic-release.gitbook.io/semantic-release/support/troubleshooting

**Reply by gr2m** [reply]
the best we can do is describe the behavior in the docs somewhere. Or maybe you could create a bitbucket plugin which explains why there are the extra commits. Using `git tag` and `git note` are core to how semantic-release works, so we cannot remove that.
